src/Pagerfanta/Adapter/ElasticaAdapter.php

```diff
@@ -72,9 +78,11 @@ public function getResultSet()
      */
     public function getSlice($offset, $length)
     {
-        return $this->resultSet = $this->searchable->search($this->query, array(
+        $options = $this->options === null ? array() : $this->options;
```

Comment:
couldn't we force options to always be an array in the constructor (defaulting to an empty array) ?